tests/integration_tests/test_catalyst.py

```diff
@@ -0,0 +1,62 @@
+try:
+    from catalyst.dl import SupervisedRunner
+except ImportError:
+    pass
```

Comment:
Question: Is there any reason to use the `try-except` statement here?

Reply:
As I think about it now, I can't see a reason to use the `try-except` statement here. I'll remove it.

Reply:
It turns out `try-except` was necessary to skip catalyst tests for python3.5 gracefully, as catalyst is only available for python version > 3.5.

Reply:
Oh, I missed it. Make sense. Your additional patch using [`pytest.importorskip`](https://docs.pytest.org/en/latest/reference.html#pytest-importorskip) is nice. Thanks for your effort in the backward compatibility!